marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/schemas/StepSchema.java

```diff
@@ -332,7 +332,7 @@ public void setHeaders(Headers headers) {
 
     /**
      * Applicable to mapping steps only
-     * 
+     *
      */
     @JsonProperty("validateEntity")
     public Boolean getValidateEntity() {
```

Comment:
You will have to update the data type in schema json file and the run 'generateJsonSchema2Pojo' task , so that it's updated from Boolean to String